docs/pages/docs/reference/command-line-reference.mdx

```diff
@@ -230,6 +230,8 @@ turbo run build --team=my-team --token=xxxxxxxxxxxxxxxxx
 
 You can also set the value of the current token by setting an environment variable named `TURBO_TOKEN`. The flag will take precedence over the environment variable if both are present.
 
+> If you're using remote caching on Vercel, you can use a Vercel Personal Access Token as your bearer token.
```

Comment:
```suggestion
If you're using Remote Caching on Vercel, and your building your project on Vercel, this environment variable and flag are not needed because they are automatically set for you. If you’re using Remote Caching on Vercel, but building in another CI provider like CircleCI or GitHub Actions, you can use a Vercel Personal Access Token as your `—token` or `TURBO_TOKEN`. If you are using a custom Remote Cache, this value will be used to send an HTTP Bearer token with requests to your custom Remote Cache.
```